drivers/hmis/app/graphql/types/hmis_schema/assessment.rb

```diff
@@ -13,6 +13,8 @@ class HmisSchema::Assessment < Types::BaseObject
 
     available_filter_options do
       arg :roles, [Types::Forms::Enums::AssessmentRole]
+      arg :project_types, [Types::HmisSchema::Enums::ProjectType]
+      arg :projects, [ID]
```

Comment:
Can you un-pluralize these (like you did for service)? `project_type` and `project`.

I opened a PR to do a whole sweep to make them consistent https://github.com/greenriver/hmis-warehouse/pull/3132

Reply:
Fixed